conversions/src/Data/Conversions.hs

```diff
@@ -225,12 +225,12 @@ convertErrorFromNatural value =
   maybe (throwError $ UserBoundError value minBound maxBound) pure
     $ checkedFromIntegral value
 
-convertErrorFromIntegral
+convertErrorBounded
```

Comment:
__suggestion:__ The git commit message references the old `convertErrorFromInt` name.